google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineSupportProvider.java

```diff
@@ -355,5 +308,10 @@ public void addSupport(@NotNull Module module,
     public JComponent createComponent() {
       return myMainPanel;
     }
+
+    @SuppressWarnings("checkstyle:abbreviationaswordinname")
```

Comment:
geez what is this checkstyle complaining about? is it because U and I are capitalized?